entsoe/entsoe.py

```diff
@@ -428,6 +430,58 @@ def query_imbalance_prices(self, country_code, start, end, psr_type=None):
         response = self.base_request(params=params, start=start, end=end)
         return response.text
 
+    def query_contracted_reserve_prices(self, country_code, start, end,
+                                        type_marketagreement_type, psr_type = None):
```

Comment:
Can you update the docstring for `type_marketagreement_type`?
What values are accepted here? Perhaps a good idea to include them in the docs or `mappings.py`

Reply:
Added definition in mappings.py